crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -257,6 +268,32 @@ static void GetTrace(std::vector<Event> *out_trace, unsigned flags,
         break;
       }
 
+      case __NR_nanosleep: {
+        // First true condition: In FIPS mode, an |ioctl| call with command
+        // |RNDGETENTCNT| is used. If this fails, a delay is injected. The
+        // failure happens when the test flag |URANDOM_NOT_READY| is set. But
+        // since this bit is cleared below we detect this event using
+        // |urandom_not_ready_was_cleared|.
+        //
+        // Second true condition: We can have two or more consecutive
+        // |nanosleep| calls. This happens if |nanosleep| returns -1. The PRNG
+        // model only accounts for one |nanosleep| call. Do the same here.
+        if (urandom_not_ready_was_cleared ||
+            ((flags & URANDOM_ERROR) && (previous_syscall != __NR_nanosleep))) {
+          out_trace->push_back(Event::NanoSleep());
+        }
+        break;
+      }
+
+      // Alias for |__NR_nanosleep| on, at least, Ubuntu 20.04.
+      case __NR_clock_nanosleep: {
```

Comment:
Can't we put this `case` with the previous one? Or is `__NR_nanosleep` in L.291 a typo and should be `__NR_clock_nanosleep`?

Reply:
Typo...